### DISCUSSION TITLE
Microphone

### DISCUSSION THREAD

**Discussion by Bruno21** [avantrec/soco-cli#9] [Q&A] [answered]
Hello,

Is it possible to activate/desactivate the microphone ?
I'm using Sonos One (S2)

**Comment by pwt** [comment, marked as answer]
Hi Bruno:

Good question. My understanding is that the microphones on applicable Sonos speakers are under hardware control only, meaning there's no software mechanism -- even for the Sonos apps -- for enabling and disabling them.

I believe this is a privacy feature, to prevent a microphone from being surreptitiously enabled without using the relevant button on a speaker.

Peter

**Comment by Bruno21** [comment]
This seems to be a good reason, although it would have been practical to activate it without getting up from your chair...

Thanks for your answer Peter.
